pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -560,6 +562,32 @@ func (p *DNSProxy) CheckAllowed(endpointID uint64, destPort uint16, destID ident
 	return false, nil
 }
 
+func configureConnection(conn *net.Conn, sec_id identity.NumericIdentity) error {
+
+	var file *os.File
+	var err error
+
+	switch l4conn := (*conn).(type) {
+	case *net.UDPConn:
+		if file, err = l4conn.File(); err != nil {
+			return fmt.Errorf("can't get file from %v: %v", l4conn, err)
```

Comment:
Error strings should be formatted with `%w`: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Thanks, didn't know. Only worked with Go-1.12 before :-)